src/pages/docs/infrastructure/deployment-targets/machine-policies.mdx

```diff
@@ -149,6 +150,32 @@ You can select a username/password account to perform automatic Tentacle updates
 
 **Note:** This option can not be used when Tentacle is running as Local System.
 
+## Recover from communication errors with Tentacle \{#MachinePolicies-Recoverfromcommunicationerrorswithtentacle}
+Configure whether Octopus Deploy should re-attempt failed communication with a Tentacle for a set duration.
+
+With this setting, if a network error occurs while Octopus Deploy is communicating with a Tentacle,
+- **When disabled** the deployment, runbook run or health check will fail.
+- **When enabled** it will re-attempt communication with the Tentacle for the configured duration. However, continued network errors will cause a failure.
```

Comment:
```suggestion
With this setting, if a network error occurs while Octopus Deploy is communicating with a Tentacle,
- **When disabled**, the deployment, runbook run, or health check will fail.
- **When enabled**, it will re-attempt communication with the Tentacle for the configured duration. However, continued network errors will cause a failure.
```